app/Models/Route.php

```diff
@@ -23,7 +44,6 @@ class Route extends DeviceRelatedModel
         'ciscoIgrp',
         'bbnSpfIgp',
         'ospf',
-        'ospfv3',
```

Comment:
This is a regression from #17046 that I just noticed. I am adding the fix to your PR instead of creating a new one if that's OK with you.

Reply:
yes, it is OK